google/cloud/bigtable/examples/README.md

```diff
@@ -61,7 +73,7 @@ $ GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=$HOME/google-cloud-cpp/third_party/grpc/etc/r
 $ export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH
 ```
 
-### Authenticate with Google Cloud
+#### Authenticate with Google Cloud Platform
```

Comment:
We should check if this step is necessary.

Reply:
@coryan 
the answer is Yes and No.
Yes because a new user must login to Google Cloud Platform and this step opens up browser for authentication and help users to download credential file (ADC file). It is not a mandatory step but helps. 
https://cloud.google.com/docs/authentication/getting-started)  

No because if user has credential file then we just need to set couple of variables to connect to Bigtable.
export GOOGLE_APPLICATION_CREDENTIALS=<path-to-json-credential-file> 
export GRPC_DEFAULT_SSL_ROOTS_FILE_PATH=<path-to-certificate-file>

So I would suggest to keep this step.